proto/api/v2/common.proto

```diff
@@ -11,3 +11,29 @@ message ResourceByID {
 
 message Empty {
 }
+
+message Schedule {
+    enum IntervalType {
+        UNSET           = 0;
+        WEEKLY          = 1;
+        MONTHLY         = 2;
+    }
+
+    // Sunday = 1, Monday = 2, .... Saturday =  7
+    message DaysOfWeek {
+        repeated int32 days = 1;
+    }
+    // 1 for 1st, 2 for 2nd .... 31 for 31st
+    message DaysOfMonth {
+        repeated int32 days = 1;
+    }
+
+    IntervalType interval_type      = 1;
```

Comment:
My hesitation with altering that is that this is a copy of `ReportSchedule` that I just put in common figuring that we could then migration `ReportSchedule` to this one.  If we do that it would have the potential to break the API by removing a field that is simply redundant.  Thoughts?

Reply:
hmm, I believe if I am to ask to change this, I risk annoying @sachaudh  and @charmik-redhat. It is bad that we already have extra baggage in `v2`.